src/lib.rs

```diff
@@ -165,6 +167,20 @@ pub enum Op {
 
     /// Denote the end of execution. All execution must end with `Done`
     Done,
+
+    // XXX These operations live at the bottom instead of with other arithmetic
```

Comment:
Let's nix the XXX and bump the micro of HIF's version.

Reply:
Done, merging!